cdap-data-fabric/src/main/java/co/cask/cdap/data/runtime/DataSetsModules.java

```diff
@@ -119,7 +119,7 @@ protected void configure() {
 
   @Override
   public Module getDistributedModules() {
-    return getDistributedModules(false);
+    return getDistributedModules(true);
```

Comment:
Publishing enabled is still controlled by CConf. With this change and having publishing disabled results in no metadata updates being published.